### PR TITLE
Change type of block size to bigint

### PR DESCRIPTION
**Pull request by JonoPrest**
I seem to have missed this field as it should be a bigint

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -224,14 +224,72 @@ switch (platform) {
         }
         break
       case 'arm':
+        if (isMusl()) {
```
Comment (ozgrakkurt) [inline index.js:227]: not sure why these changes are happening, do you run `yarn update` or something similar?

Comment (JonoPrest) [reply index.js:227]: @ozgrakkurt do you not get the same if you run `yarn build` or `yarn build:debug`?
